configure.ac

```diff
@@ -526,13 +526,10 @@ else
   # Sphinx exits with code 1 when it prints its usage
   sphinxver=`{ $SPHINX --version 2>&1 || :; } | head -n 3`
 
-  if ! echo "$sphinxver" | grep -q -w -e '^Sphinx' -e '^Usage:'; then
```

Comment:
This check does not need to go, it just needs a `-i` in grep's invocation.

Reply:
Fixed.